modules/ROOT/pages/planning-and-tuning/query-tuning.adoc

```diff
@@ -303,3 +303,28 @@ CYPHER replan=force EXPLAIN MATCH ...
 
 During times of known high load, `replan=skip` can be useful to not introduce unwanted latency spikes.
 
+[[cypher-infer-schema-parts]]
+=== Cypher infer schema parts
+
+For some queries, the planner can infer predicates such as labels or types from the graph structure that can improve estimating the number of rows that each operator produces.
+(See xref:planning-and-tuning/execution-plans.adoc#runtimes-reading-execution-plans[Cypher Manual -> Execution plans and query tuning -> Understanding execution plans])
+This option allows to control to what extent it should do that:
```

Comment:
```suggestion
For some queries, the planner can infer predicates such as labels or types from the graph structure, thereby enhancing its ability to estimate the number of rows each operator will produce.
(See xref:planning-and-tuning/execution-plans.adoc#runtimes-reading-execution-plans[Understanding execution plans - Reading execution plans] for more information about the role of operators in query execution plans.)
The option `inferSchemaParts` controls the extent to which the planner should infer predicates.
```

Reply:
I think your version sounds a lot better. But I especially wanted to reference our only documentation on estimated rows.

What do you think about
```suggestion
For some queries, the planner can infer predicates such as labels or types from the graph structure, thereby enhancing its ability to estimate the number of rows each operator will produce.
(See xref:planning-and-tuning/execution-plans.adoc#runtimes-reading-execution-plans[Understanding execution plans - Reading execution plans] for more information about the role of operators and estimated row counts in query execution plans.)
The option `inferSchemaParts` controls the extent to which the planner should infer predicates.
```

Reply:
Sounds good 👍